mobile/lib/screens/dashboard/dashboard_view.dart

```diff
@@ -302,4 +360,11 @@ class _DashboardViewState extends State<DashboardView> {
     context.read<NearbyLocationBloc>().add(const SearchNearbyLocations());
     await _appService.refreshDashboard(context);
   }
+
+  Future<void> showcasetoggle() async {
+    final prefs = await SharedPreferences.getInstance();
+    if (prefs.getBool('showcase') == null) {
+      _startShowcase();
+    }
+  }
```

Comment:
Method to target new devices, or devices which have never experienced the showcase feature.